examples/rpc/rpc_from_drg_module/main.tf

```diff
@@ -15,141 +15,143 @@ terraform {
 
 # Resources
 
-module "vcn_local" {
-  # this module use the generic vcn module and configure it to act as local vcn
-  source  = "oracle-terraform-modules/vcn/oci"
-  
+module "vcn_acceptor" {
+  # this module use the generic vcn module and configure it to act as vcn for RPC acceptor
+  source = "oracle-terraform-modules/vcn/oci"
+
```

Comment:
The version of the module is also required

Reply:
The more I think about pinning version in modules examples or not, the more I tend to say no. Here is the rational behind my current view on that matter.

The problem is that the example's files are not released yet, hence version 3.2.0 will not be able to support this example: user will get an error when trying to copy/paste and execute the example.

In addition, we would always need to come back to example files and keep module version current.

Letting modules in examples without versions on purpose seems to be a better tradeoff:
- less maintenance,
- released versions on the registry (that is, with a tag) always works.
- bleeding edge users can always source the module using the git pattern url and use repo's `HEAD` (great for testing by the way).

But let's settle this later and not block the PR from being merged into `main`. If we choose to not pin module versions in examples, we will to search and update every occurrences of it before cutting the next release tag.